src/components/messaging/conversationInterlocutor.component.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {
+  View,
+  ViewProps,
+} from 'react-native';
+import { Profile } from '@src/core/model';
+import { Avatar } from '@kitten/ui';
+import {
+  ThemedComponentProps,
+  ThemeType,
+  withStyles,
+} from '@kitten/theme';
+
+interface ComponentProps {
+  profile: Profile;
+}
+
+export type ConversationInterlocutorProps = & ThemedComponentProps & ViewProps & ComponentProps;
+
+class ConversationInterlocutorComponent extends React.Component<ConversationInterlocutorProps> {
+
+  private renderOnlineIndicator = (): React.ReactElement<ViewProps> => {
```

Comment:
return type can be null, so I suggest you do null check before calling instead of using control fllow in `renderSomething` functions

Reply:
Refactored.